lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -814,38 +798,6 @@ unpack_simple_colocation(xmlNode *xml_obj, const char *id,
         pcmk__config_err("Ignoring constraint '%s' because resource '%s' "
                          "does not exist", id, primary_id);
         return;
-
-    } else if ((dependent_instance != NULL) && !pcmk__is_clone(dependent)) {
-        pcmk__config_err("Ignoring constraint '%s' because resource '%s' "
-                         "is not a clone but instance '%s' was requested",
-                         id, dependent_id, dependent_instance);
-        return;
-
-    } else if ((primary_instance != NULL) && !pcmk__is_clone(primary)) {
-        pcmk__config_err("Ignoring constraint '%s' because resource '%s' "
-                         "is not a clone but instance '%s' was requested",
-                         id, primary_id, primary_instance);
-        return;
-    }
-
-    if (dependent_instance != NULL) {
-        dependent = find_clone_instance(dependent, dependent_instance);
```

Comment:
There are no more callers of `find_clone_instance()`

Reply:
thx